.github/workflows/optimize-ci.yml

```diff
@@ -439,30 +360,33 @@ jobs:
       - name: Read Java/version info
         id: "pom-info"
         uses: YunaBraska/java-info-action@main
-      - name: Start Elasticsearch
+      - name: Start Database
         uses: ./.github/actions/compose
         with:
-          compose_file: .github/actions/compose/docker-compose.elasticsearch.yml
-          project_name: elasticsearch
+          compose_file: .github/actions/compose/docker-compose.${{ matrix.database }}.yml
+          project_name: ${{ matrix.database }}
         env:
           ELASTIC_VERSION: ${{ steps.pom-info.outputs.x_elasticsearch_test_version }}
           ELASTIC_JVM_MEMORY: 1
           ELASTIC_HTTP_PORT: ${{ steps.pom-info.outputs.x_new_elasticsearch_port }}
+          OPENSEARCH_VERSION: ${{ steps.pom-info.outputs.x_opensearch_test_version }}
+          OPENSEARCH_JVM_MEMORY: 1
+          OPENSEARCH_HTTP_PORT: ${{ steps.pom-info.outputs.x_new_elasticsearch_port }}
       - name: Install
         uses: ./.github/actions/run-maven
         with:
-          parameters: install -Dskip.fe.build -Dskip.docker -DskipTests -pl optimize -am
+          parameters: install -Dskip.fe.build -Dskip.docker -DskipTests -pl optimize/upgrade -am
```

Comment:
@PHWaechtler , this is the fix that would also solve https://github.com/camunda/camunda-optimize/issues/14153

Reply:
Nice! I have linked the PR to that ticket too. Can you pick this over to C7 too or is this already working there?

Reply:
From the ticket it was not clear to me which pipelines exactly are affected by this. All I can tell you is that whichever pipeline is affected, this is the fix. Maybe this was the only one to begin with and this PR already closes that ticket?